assets/info-page/index.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script>
+      var mattermostPrimaryPlatform = 'mattermost';
+      var msteamsPrimaryPlatform = 'msteams'
+      var primaryPlatform = mattermostPrimaryPlatform;
+
+      function closeInfoPage() {
+        window.close();
+      }
+
+      function applyBorder(primaryPlatform, secondaryPlatform) {
+        document.getElementById(primaryPlatform).setAttribute(
+          'style',
+          'outline: 3px solid #1C58D9;'
+        )
+
+        document.getElementById(secondaryPlatform).setAttribute(
+          'style',
+          'outline: 1px solid #3F435029;'
+        )
+      }
+
+      function showTickIcon(primaryPlatform, secondaryPlatform) {
+        document.getElementById(primaryPlatform).setAttribute(
+          'style',
+          'display: block;'
+        )
+
+        document.getElementById(secondaryPlatform).setAttribute(
+          'style',
+          'display: none;'
+        )
+      }
```

Comment:
we can rather add two classes `d-block` and `d-none` with required styles and toggle them instead adding inline styles on the target elements.